GSoC-2025/brainglobe-cellfinder-improve-cellfinder's-classification-algorithm(FatmaElsharkawy).md

```diff
@@ -0,0 +1,137 @@
+## Project title
+
+`brainglobe-cellfinder:improve-cellfinder's-classification-algorithm (FatmaElsharkawy)` 
+
+## Personal details
+Please include the following information:
+- **Full name** Fatma Elsharkawy
+- **Email** fatma.elsharkawy03@eng-st.cu.edu.eg
+- **GitHub username** FatmaElsharkawy
+- **Zulip username** FatmaElsharkawy
+- **Location & time-zone** Cairo (GMT+2)
+- **Personal website / project portfolio** [Resume](https://drive.google.com/file/d/1bc4Yg1LuYDoU28bitd9_uP_SvJfGp0Kz/view?usp=sharing)
+- **Code contribution**
+    [Fix progress bar text in cell detection widget](https://github.com/brainglobe/cellfinder/pull/510)
+
+- **Proposal discussion link**
+    [#43: brainglobe-cellfinder:improve-cellfinder's-classification-algorithm (FatmaElsharkawy)](https://github.com/neuroinformatics-unit/gsoc/pull/43) 
+
+## Project proposal 
+
+- **Synopsis**
+
+    This project aims to enhance the deep learning classifier in the BrainGlobe cellfinder tool, which detects cells in large-scale brain microscopy images. Currently, cellfinder uses a ResNet-based model for classifying cell candidates, but newer architectures may offer better accuracy, speed, or efficiency. The goal is to implement and evaluate modern alternatives to ResNet, ensuring cellfinder remains at the cutting edge of cell detection. 
+    
+    **It's important because** other DL techniques may outperform the current architecture making cell detection and classification faster, robust, and hence more reliable by neuroscientists. With the very fast development of DL approaches, cellfinder has to be up-to-date to add a true value for neuroscience research. detecting and mapping cells in 3D throughout the entire mammalian brain is crucial for faster brain understanding and developments in neuroscience.
```

Comment:
Minor point, but could you expand on how new architectures will make it more robust and reliable?